react_app/src/App.js

```diff
@@ -1,35 +1,156 @@
+import Clock from "./Clock";
+import Card from './Cards.js'
+import React, { useState} from "react";
+
+const api = {
+  key: "af103c190cd36ff3f3fb1e0c135a2ee1",
+  base: "https://api.openweathermap.org/data/2.5/"
+}
 
-import { Component } from 'react';
-import Navbar from "./Components/Navbar/Navbar";
-import { Jumbotron } from 'reactstrap';
-import './App.css';
 
-let apikey = "af103c190cd36ff3f3fb1e0c135a2ee1";
 
+function App() {
 
-class App extends Component{
+  const [query, setQuery] = useState('');
+  const [weather, setWeather] = useState({});
+  const [weather2, setWeather2] = useState({});
```

Comment:
give more meaningful name. Like weather and detailed weather. Weather 2 makes me feel like you pulled for a different location

Reply:
done